package.json

```diff
@@ -52,30 +52,30 @@
     "@types/mdast": "^3.0.10",
     "@typescript-eslint/eslint-plugin": "^5.3.0",
     "@typescript-eslint/parser": "^5.3.0",
-    "babel-jest": "^28.1.1",
     "commitlint": "^14.1.0",
     "eslint": "^8.1.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
     "husky": "^7.0.4",
     "jest": "^27.5.1",
+    "jest-light-runner": "^0.5.0",
     "jest-specific-snapshot": "^5.0.0",
-    "prettier": "^2.6.1",
+    "prettier": "^3.0.0",
     "rollup": "^2.70.1",
     "standard-version": "^9.3.2",
     "terser": "^5.12.1",
-    "ts-jest": "^27.1.4",
+    "ts-node": "^10.9.1",
     "typescript": "^4.4.4"
   },
   "peerDependencies": {
-    "prettier": ">=2.1.2"
+    "prettier": ">=3.0.0"
```

Comment:
Please do not make the same mistake as your predecessors. The peer dependency should be declared with `^` or `~`, not `>=` ❌ 

Also, it would be fair to correct the peer dependency on _prettier_ of the current _0.4_ line of this plugin to read `^2.1.2`